src/Steps/Release/MergeUpRelease.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace SilverStripe\Cow\Steps\Release;
+
+use SilverStripe\Cow\Model\Release\LibraryRelease;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Merge up released commits to the major branch and optionally remove x.y-release branch.
+ * E.g. if you just released 4.12.0, this will merge 4.12-release up to 4.12, and then 4.12
+ * up to 4 - and then optionally delete the 4.12-release branch.
+ */
+class MergeUpRelease extends ReleaseStep
+{
+    public function getStepName()
+    {
+        return 'mergeup';
+    }
+
+    public function run(InputInterface $input, OutputInterface $output)
+    {
+        $this->log($output, "Merging up all modules");
+
+        $this->mergeRecursive($output, $this->getReleasePlan(), $input->getOption('delete-release-branch') ?? false);
```

Comment:
This is unnecessary. `delete-release-branch` is a `InputOption::VALUE_NONE` type option - which means it can only be true or false. I think the null coallescing operator there is redundant as well.